kasa/tests/conftest.py

```diff
@@ -346,7 +345,7 @@ def device_for_file(model, protocol):
     if protocol == "SMART":
         for d in PLUGS_SMART:
             if d in model:
-                return SmartPlug
+                return SmartDevice
```

Comment:
We can get rid of separate construction of smartdevices as soon as SmartBulb gets removed.